.gitignore

```diff
@@ -7,8 +7,13 @@ __pycache__/
 .virtualenv/
 .venv/
 
-# Local user data (settings, database, media, search index, static files)
```

Comment:
Please readd these lines. They are required for Plugin development with virtualenv installed OpenSlides (core).